### ISSUE TITLE
New Zealand vaccination data missing ages 5-11

### ISSUE THREAD

**Issue by wmtgts** [owid/covid-19-data#2295]
NZ records the vaccination data for ages 5-11 in a separate table on the MoH website, which is the source used by newzealand.py OWID is only showing data for vaccinations given to ages 12+. The latest OWID contains is listed below, this is missing the 51639 5-11 year olds

```
location	iso_code	date	total_vaccinations
New Zealand	NZL	18/01/2022	8757497
New Zealand	NZL	19/01/2022	
New Zealand	NZL	20/01/2022	8840466
```

![image](https://user-images.githubusercontent.com/70423883/150610079-74b8b2a0-bb3f-412e-a8ae-0a26173fcc9f.png)

**Comment by lucasrodes** [comment]
Thanks, @wmtgts,
We have added vaccination data for 5-11. This currently only applies to the latest point, as we wait for NZ to make this available in [their vaccination excel file](https://www.health.govt.nz/system/files/documents/pages/covid_vaccinations_18_01_2022_0.xlsx).

**Comment by MarrickLip** [comment]
@lucasrodes I noticed today that paediatric first doses still aren't being included. [Per the Ministry](https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-data-and-statistics/covid-19-vaccine-data), there have been 4,006,972 first doses in 12+ and 183,727 for ages 5-11. However, in Our World in Data, the "[Number of people who received at least one dose of COVID-19 vaccine](https://ourworldindata.org/explorers/coronavirus-data-explorer?zoomToSelection=true&time=2020-03-01..latest&facet=none&pickerSort=asc&pickerMetric=location&Metric=People+vaccinated&Interval=Cumulative&Relative+to+Population=false&Color+by+test+positivity=true&country=NZL)" shows New Zealand at 4.01 million i.e. just 12+.

P.S. in NZ, the first children will be eligible for their second doses on February 7th so the Ministry's web page will likely change around then to count these as well.

**Comment by edomt** [comment]
@lucasrodes (See also https://github.com/owid/covid-19-data/commit/e3c53f29a0d6356db2ff4dae902589347f0aeb65#commitcomment-65907110 - it seems the new batch file published by NZ indeed has much lower figures than a few days ago)

**Comment by lucasrodes** [comment]
Hi,
Thanks for raising this up. 

TL;DR: I will be fixing this shortly!

For some context, we are currently having some difficulties in collecting and producing the dataset for NZ. NZ reports the data in two ways:

- [Full-time series](https://www.health.govt.nz/system/files/documents/pages/covid_vaccinations_01_02_2022.xlsx): Which is very useful to keep track of backwards correction in the data. However, this data _does not include_ <12 year old persons.
- [Single latest value](https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-data-and-statistics/covid-19-vaccine-data): Which provides the _latest_ available figures for <12.

So far we have been reporting data for _all_ population for the single last entry, while the previous dates only have included >12 data. However, something went wrong yesterday and also only >12 data was shown for the latest date.

**Comment by lucasrodes** [comment]
I have modified our process to account for the history of 5-12 vaccination data. 

I will be monitoring this in the upcoming days as the process might not be stable. I hope NZ soon adds <12 data to the main [excel file](https://www.health.govt.nz/system/files/documents/pages/covid_vaccinations_01_02_2022.xlsx).

> P.S. in NZ, the first children will be eligible for their second doses on February 7th, so the Ministry's web page will likely change around then to count these as well.

Thanks for letting us know, @MarrickLip!

**Comment by USMortality** [comment]
>     * [Full-time series](https://www.health.govt.nz/system/files/documents/pages/covid_vaccinations_01_02_2022.xlsx): 
The links mentioned here, as well as the excesl files, are broken/removed by NZL, any chance there's a backup anywhere? Thx!
